From e52fe9f92e3e063b45b37a1990b99df363e86e5d Mon Sep 17 00:00:00 2001
From: Hou Qi <qi.hou@nxp.com>
Date: Wed, 17 Aug 2022 16:57:11 +0800
Subject: [PATCH 01/19] V4L2VDA: Switch to use VDA instead of direct
 VideoDecoder

commit b7f3a9e8058f593d7d88b6b6cafa71957aa3f1a1 aims to default
enable "direct VideoDecoder" support on Linux. So need to switch
to use VDA path iff:

1. kVaapiVideoDecodeLinux is enabled (disabled by default);
2. kUseChromeOSDirectVideoDecoder is disabled (enabled by default);
3. GL is used;

Upstream-Status: Inappropriate [NXP specific]
---
 media/base/media_switches.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/media/base/media_switches.cc b/media/base/media_switches.cc
index bb277d181b482..f99e39ec25fae 100644
--- a/media/base/media_switches.cc
+++ b/media/base/media_switches.cc
@@ -1134,7 +1134,7 @@ BASE_FEATURE(kChromeOSHWVBREncoding,
 // TODO(b/159825227): remove when the direct video decoder is fully launched.
 BASE_FEATURE(kUseChromeOSDirectVideoDecoder,
              "UseChromeOSDirectVideoDecoder",
-             base::FEATURE_ENABLED_BY_DEFAULT);
+             base::FEATURE_DISABLED_BY_DEFAULT);
 
 // Limit the number of concurrent hardware decoder instances on ChromeOS.
 BASE_FEATURE(kLimitConcurrentDecoderInstances,
-- 
2.34.1

